scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -83,6 +83,51 @@ def process_response(self, request, response, spider):
 
         return response
 
+    def _handle_encoding(self, body, content_encoding):
+        content_to_decode = body
+
+        separator = b","
+        decoded_body = ""
+        for index, encodings in reversed(list(enumerate(content_encoding))):
+            if separator in encodings:
+                decoded_body = ""
```

Comment:
I think this line is unnecessary, and would break scenarios with multiple headers where some of those headers include comma-separated encodings.